cmd/thanos/rule.go

```diff
@@ -347,3 +503,13 @@ func parseFlagLabels(s []string) (labels.Labels, error) {
 	}
 	return lset, nil
 }
+
+func labelsTSDBToProm(lset labels.Labels) (res promlabels.Labels) {
```

Comment:
yet another convert function for labels ;p I lost counting, how many of these we have ^^

Reply:
Yes, it's pretty annoying. We've largely managed to at least keep every `pkg` to convert only between two of `prometheus`, `tsdb` and `storepb` types. But in main going full circle isn't avoidable.

I thought about extracting a `convert` package. But often conversions have small nuances to it like adding a external labels while converting and such.
Looks like we'd get lost in a lot of weirdly named functions so that just doing it on-demand seems like less cognitive overhead.